frontend/grunt-tasks/grunt-watch.js

```diff
@@ -8,25 +8,17 @@ module.exports = function (grunt) {
         gruntfile: {
             files: ['Gruntfile.js']
         },
-        includeSource: {
-            files: [
-                //'<%= yeoman.app %>/**/*.html',
-                '<%= yeoman.app %>/**/{,*/}*.html',
-                '<%= yeoman.app %>/**/{,*/}*.js'
-            ],
-            tasks: ['includeSource:dev', 'copy:js']
-        },
         livereload: {
             options: {
-                livereload: '<%= connect.options.livereload %>'
+                livereload: true
```

Comment:
'<%= connect.options.livereload %>' - lets move it back, just to make sure it's configured in connect task